tests/autoscaling/test_autoscaling_service_lib.py

```diff
@@ -286,7 +286,7 @@ def test_mesos_cpu_metrics_provider():
     with mock.patch(
         'paasta_tools.utils.KazooClient', autospec=True,
         return_value=mock.Mock(get=mock.Mock(
-            side_effect=lambda x: (zookeeper_get_payload[x.split('/')[-1]], None),
+            side_effect=lambda x: (str(zookeeper_get_payload[x.split('/')[-1]]).encode('utf-8'), None),
```

Comment:
the test inputs here are sometimes ints and stuff, I wanted to make sure the mock returned what zk has exactly (bytes of a floaty string)